src/test/scala/com/microsoft/hyperspace/index/IndexLogManagerImplTest.scala

```diff
@@ -92,7 +95,7 @@ class IndexLogManagerImplTest
 
   test("testGetLog returns None if log not found") {
     val path = new Path(testRoot, "testPath")
-    assert(new IndexLogManagerImpl(path).getLog(0).isEmpty)
+    assert(logManager(path).getLog(0).isEmpty)
```

Comment:
nit: you could have just made `path` as absolute?

Reply:
thanks @imback82, done